Assets/Scripts/MessageImageDisplayer.cs

```diff
@@ -406,6 +423,13 @@ public void SetCuedRecallMessage(string bottomText, bool hospital=false, bool ec
             cued_recall_continue.transform.Find("continue text").GetComponent<Text>().text = LanguageSource.GetLanguageString(bottomText);
     }
 
+    public void DoRecallProgressDisplay(bool progressOn, float time)
```

Comment:
You can change these names from "RecallProgressDisplay" or "recall_progress" to just "ProgressDisplay" and "progress_display"

Reply:
Also, you call it a progress bar in the variables uptop. So you probably should call it a progress bar everywhere else.